xml/System.Security.Cryptography.X509Certificates/X509Certificate2.xml

```diff
@@ -1514,7 +1514,7 @@ This loads the first well-formed PEM found with a CERTIFICATE label.
             
 For PEM-encoded certificates with a private key, use <xref:System.Security.Cryptography.X509Certificates.X509Certificate2.CreateFromPem(System.ReadOnlySpan{System.Char},System.ReadOnlySpan{System.Char})>.
 
-For PEM-encoded certificates in a file, use <xref:System.Security.Cryptography.X509Certificates.X509Certificate2.%23ctor(System.String)>.
+For PEM-encoded certificates in a file, use <xref:System.Security.Cryptography.X509Certificates.X509Certificate2.CreateFromPemFile(System.String)>.
```

Comment:
Both the ctor and CreateFromPemFile work for PEM content, but they do different things.

* ctor: Reads out the certificate, ignores the private key
* CreateFromPemFile: Reads out the certificate, finds a matching private key, **fails if there is not a matching private key**

So it's probably better to break this into two different paragraphs, like


```suggestion
For PEM-encoded certificates in a file without a private key, use <xref:System.Security.Cryptography.X509Certificates.X509Certificate2.%23ctor(System.String)>.

For PEM-encoded certificates in a file with a private key, use <xref:System.Security.Cryptography.X509Certificates.X509Certificate2.CreateFromPemFile(System.String)>.
```